test/readdlm.jl

```diff
@@ -162,7 +162,7 @@ let i18n_data = ["Origin (English)", "Name (English)", "Origin (Native)", "Name
         "Ireland", "Sinead O'Connor", "Éire", "Sinéad O'Connor",
         "Israel", "Yehoram Gaon", "ישראל", "יהורם גאון",
         "Italy", "Fabrizio DeAndre", "Italia", "Fabrizio De André",
-        "Japan", "KUBOTA Toshinobu", "日本", "久保田    利伸",
+        "Japan", "KUBOTA Toshinobu", "日本", "久保田    利伸",
```

Comment:
Unless some strange regression happens that only gets noticed when run on nbsp, but I suppose that would be unlikely?